### PR TITLE
package.json: Upgrade to node-sass ~3.4.2

### LINKED ISSUE #986: Node.js v5 incompatibility
it is not possible to install bootstrap-sass 3.3.5 on a windows 8.1 system with node 5.0.0 and npm 3.4.0 node-sass 3.3.3 and lib-sass 3.2.5 (no problem with v4-dev)

-----------------------------------------------------

Can not download file from https://raw.githubusercontent.com/sass/node-sass-binaries/v2.0.1/win32-x64-node-5.0/binding.node

node-sass@2.0.1 postinstall ...\bootstrap3\node_modules\node-sass
node scripts/build.js

module.js:339
    throw err;
    ^

Error: Cannot find module '...\bootstrap3\node_modules\node-sass\node_modules\pangyp\bin\node-gyp'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:136:18)
    at node.js:972:3
 Build failed

### PR DESCRIPTION
**Pull request by cvrebert**
Fixes #986.

### CODE REVIEW

**Comment by cvrebert** [comment]
@twbs/sass Ping.

**Comment by glebm** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ gemfile:
   - test/gemfiles/sass_3_4.gemfile
   - test/gemfiles/sass_head.gemfile
 before_install:
+  - "nvm install stable"
   - "npm install"
```
Comment (glebm) [inline .travis.yml]: Do we need both install stable and install?

Comment (cvrebert) [inline .travis.yml]: `nVm` ≠ `nPm`

**Pull request merged**